packages/frontend/src/app/main/main.component.ts

```diff
@@ -39,7 +40,7 @@ export class MainComponent implements OnInit {
     this.route.params.subscribe(params => {
       const foundParams = this.interpreter.parseRouteParam(
         params,
-        expectedParams.get('main')
+        expectedParams.get(RouteProvider.routes.main.name)
```

Comment:
The idea is to hold those strings in the provider.

Reply:
Is the goal here that we could display two types of  entities in the same main component, e.g., repositories in the repo view,  and tests  in the test view?

Reply:
The goal is actually to "untie" each component from a particular route. So the routes are all defined in the RouteProvider and each component can be linked to any of them. This is actually the first step in introducing a refactor requested by @chingor13 in #64.
The idea of reusing the same component for every page is coming in another PR after Martin's input.